src/dataSources/cloudFirestore/members.js

```diff
@@ -13,7 +13,7 @@ const member = dbInstance => {
     dlog('total member count');
     return membersCollection
       .where('isDeactivated', '==', false)
-      .select('firstName')
+      .select()
```

Comment:
note for the future.. We could get better about reading the fields from the AST that are being asked for in the query and spreading them here.